ui/widgets/nuxeo-operation-button.js

```diff
@@ -235,6 +283,87 @@ import '../actions/nuxeo-action-button-styles.js';
     _computeTooltip(tooltip, label) {
       return tooltip || label;
     }
+
+    _isPageProviderDisplayBehavior(input) {
+      return (
+        input &&
+        input.behaviors &&
+        Nuxeo.PageProviderDisplayBehavior &&
+        Nuxeo.PageProviderDisplayBehavior.every((p) => input.behaviors.includes(p))
+      );
+    }
+
+    _isSelectAllActive(input) {
+      return this._isPageProviderDisplayBehavior(input) && input.selectAllActive;
+    }
+
+    _isRunning(status) {
+      if (status['entity-type'] === 'bulkStatus') {
+        const { state } = status.value ? status.value : status;
+        return state !== 'ABORTED' && state !== 'COMPLETED';
+      }
+      return status === 'RUNNING';
+    }
+
+    _isAborted(status) {
+      if (status['entity-type'] === 'bulkStatus') {
+        const { state } = status.value ? status.value : status;
+        return state === 'ABORTED';
+      }
+      return this._isRunning(status);
+    }
+
+    _hasBulkStatus(status) {
+      return status['entity-type'] === 'bulkStatus';
```

Comment:
Should be cool if `status` is always defined.

Reply:
What do you mean? `status` should always be defined indeed. But do you think I should do something like: 

```
return status && status['entity-type'] === 'bulkStatus';
```

just to be sure?

Reply:
Exactly, but if know it won't happen maybe it's not worthing checking.

Reply:
I agree that all of these methods checking on status should check if status is defined first, just in case.